### ISSUE TITLE
Tab-order should persist when navigating to vertical from universal search

### ISSUE THREAD

**Issue by b1lly** [yext/answers-search-ui#26]
The tab order should be persisted for the Navigation Component when navigating from Universal Results to Vertical results and so forth.

Right now, during Universal Results, the Navigation Component has it's ordering dynamically updated based on the results/server response. When navigating from that page to a specific vertical, the navigation will change to whatever the navigation is configured to be.

In order to persist the TabOrder from the origin, we should append the "TabOrder" object to the URL as params on each of the links within the navigation component. Then, when the navigation component loads, the first act could be to retrieve it from the URL params if exists and use it as it's default.

**Comment by b1lly** [comment]
Fixed in #54
